envault/vault.py

```diff
@@ -30,3 +33,16 @@ def get_secrets(server_uri, secrets_path, token):
     except HTTPError as e:
         raise SystemExit("Error: " + str(e))
 
+def get_profile_configs(profile = 'default'):
```

Comment:
`vault.py` is used as a service layer to only perform vault specific tasks.
This however is a configuration specific to envault itself. Add this logic to `config.py`